### PR TITLE
feat: adding support and samples for Jsonb

### PR DESCRIPTION
**Pull request by asthamohta**

### CODE REVIEW

File: samples/samples/pg_snippets.py [hunk 1]
```diff
@@ -1330,27 +1321,133 @@ def query_data_with_query_options(instance_id, database_id):
...
 
         for row in results:
             print("VenueId: {}, VenueName: {}, LastUpdateTime: {}".format(*row))
     # [END spanner_postgresql_query_with_query_options]
 
 
+# [START spanner_postgresql_jsonb_add_column]
+def add_jsonb_column(instance_id, database_id):
+    """Adds a new JSONB column to the Venues table in the example database."""
+    spanner_client = spanner.Client()
```
Comment (kurtisvg) [inline samples/samples/pg_snippets.py:1339]: Please include the following comment on client initializations: https://googlecloudplatform.github.io/samples-style-guide/#clients

File: samples/samples/pg_snippets.py [hunk 2]
```diff
@@ -1364,92 +1461,90 @@ def query_data_with_query_options(instance_id, database_id):
...
+    subparsers.add_parser("update_data_with_dml", help=update_data_with_dml.__doc__)
+    subparsers.add_parser("delete_data_with_dml", help=delete_data_with_dml.__doc__)
     subparsers.add_parser(
-      "dml_write_read_transaction", help=dml_write_read_transaction.__doc__
+        "dml_write_read_transaction", help=dml_write_read_transaction.__doc__
     )
     subparsers.add_parser("insert_with_dml", help=insert_with_dml.__doc__)
     subparsers.add_parser(
-      "query_data_with_parameter", help=query_data_with_parameter.__doc__
+        "query_data_with_parameter", help=query_data_with_parameter.__doc__
```
Comment (kurtisvg) [inline samples/samples/pg_snippets.py:1495]: CLIs for samples are strongly discouraged: https://googlecloudplatform.github.io/samples-style-guide/#clients

If we are adding new samples it would be better to put them in their own file and follow the existing best practices

File: samples/samples/pg_snippets.py [hunk 3]
```diff
@@ -38,19 +39,19 @@ def create_instance(instance_id):
     spanner_client = spanner.Client()
 
     config_name = "{}/instanceConfigs/regional-us-central1".format(
-      spanner_client.project_name
+        spanner_client.project_name
```
Comment (kurtisvg) [inline samples/samples/pg_snippets.py:42]: Please avoid making changes to other snippets and samples outside of the PR. If their needs to be a lint action to get existing things clean, please do that in it's own PR.

**Review by kurtisvg — CHANGES_REQUESTED** [review]

**Review by rajatbhatta — APPROVED** [review]

**Review by rajatbhatta — COMMENTED** [review]
LGTM, except please fix the samples lint errors, before merging.

**Review by ansh0l — APPROVED** [review]
lgtm

File: samples/samples/pg_snippets.py (hunk 3 above)
Comment (ansh0l) [reply samples/samples/pg_snippets.py]: Sure @kurtisvg. We'll keep that in mind next time onwards. 

Can you take a look if there's anything else pending, since we are looking to merge these later today.
